src/routes/dockerflow.js

```diff
@@ -27,6 +27,11 @@ export function dockerFlowRoutes() {
     // We try to get the version file in the current working directory.
     const versionFilePath = 'version.json';
     try {
+      const stat = await fs.promises.stat(versionFilePath);
+      ctx.lastModified = stat.mtime;
+
+      // We could try to conditionally answer depending on cache-related headers
+      // in the request but this is a lot of work for this small file.
```

Comment:
Also there are some ready-to-use middlewares to do all this, but I decided this wasn't worth it yet. Later if we need it to output more serious static resources we could revisit.